recipes/liburing/all/conanfile.py

```diff
@@ -62,7 +66,10 @@ def _configure_autotools(self):
             return self._autotools
 
         self._autotools = AutoToolsBuildEnvironment(self)
-        self._autotools.configure()
+        args = []
+        if self.options.get_safe("with_libc") is False:
```

Comment:
```suggestion
        if self.options.get_safe("with_libc", False) == False:
```
Because you want to compare literal values, not a type. Also, it returns `None` by default, if not explicit.